metrics_sdk/lib/opentelemetry/sdk/metrics/aggregation/histogram_data_point.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Metrics
+      module Aggregation
+        HistogramDataPoint = Struct.new(:attributes,            # optional Hash{String => String, Numeric, Boolean, Array<String, Numeric, Boolean>}
+                                        :start_time_unix_nano,  # optional Integer nanoseconds since Epoch
+                                        :time_unix_nano,        # Integer nanoseconds since Epoch
+                                        :count,                 # Integer count is the number of values in the population. Must be non-negative.
+                                        :sum,                   # Integer sum of the values in the population. If count is zero then this field then this field must be zero
+                                        :bucket_counts,         # optional Array[Integer] field contains the count values of histogram for each bucket.
+                                        :explicit_bounds,       # Array[Float] specifies buckets with explicitly defined bounds for values.
+                                        :exemplars,             # optional List of exemplars collected from measurements that were used to form the data point
+                                        :flags,                 # Integer Flags that apply to this specific data point.
```

Comment:
What are the flags and how are they set?

Reply:
I just mirrored the OTLP definitions when creating this structs, so they're not being used or set by any code within the project right now.

https://github.com/open-telemetry/opentelemetry-proto/blob/v0.19.0/opentelemetry/proto/metrics/v1/metrics.proto#L314-L330

```
// DataPointFlags is defined as a protobuf 'uint32' type and is to be used as a
// bit-field representing 32 distinct boolean flags.  Each flag defined in this
// enum is a bit-mask.  To test the presence of a single flag in the flags of
// a data point, for example, use an expression like:
//
//   (point.flags & FLAG_NO_RECORDED_VALUE) == FLAG_NO_RECORDED_VALUE
//
enum DataPointFlags {
  FLAG_NONE = 0;

  // This DataPoint is valid but has no recorded value.  This value
  // SHOULD be used to reflect explicitly missing data in a series, as
  // for an equivalent to the Prometheus "staleness marker".
  FLAG_NO_RECORDED_VALUE = 1;

  // Bits 2-31 are reserved for future use.
}
```

Reply:
Do we anticipate producing the "no recorded value" flag? If not, I'd remove the `:flags` for now.

Reply:
Removed d3e5eea855b9a60a7f0233374fdb97685d5849a9